extensions/exegol/src/utils/cli.tsx

```diff
@@ -0,0 +1,125 @@
+import { exec } from "child_process";
+import { promisify } from "util";
+import { ExegolContainer } from "../models/ExegolContainer";
+import { readFile, writeFile } from "fs/promises";
+import { join } from "path";
+import { homedir } from "os";
+import { getPreferenceValues } from "@vicinae/api";
+import { existsSync } from "fs";
+import { DEFAULT_CONFIG } from "./config";
+
+const execPromise = promisify(exec);
+
+const EXEGOL_PROFILES_PATH = "/.exegol/profiles.json";
+
+export async function parseExegolProfiles(): Promise<Object> {
+	const profileFilename = join(homedir(), EXEGOL_PROFILES_PATH);
+
+	try {
+		if (!(await existsSync(profileFilename))) {
+			console.log("The 'profiles.json' doesn't exist. Creating it...");
+			await writeFile(profileFilename, DEFAULT_CONFIG);
+		}
+
+		var profileData = await readFile(profileFilename, "utf-8");
+
+		return JSON.parse(profileData);
+	} catch (error) {
+		console.log(error);
+		return {};
+	}
+}
+
+export function constructCommandFromExegolProfiles(
+	profileData: { [key: string]: any },
+	profileName: String,
+): String[] {
+	var cmd = [];
+
+	for (var parameter in profileData[profileName]) {
+		var parameterValue = profileData[profileName][parameter];
+
+		if (parameter === "image") {
+			continue;
+		}
+
+		if (typeof parameterValue === "string" && parameterValue !== "") {
+			cmd.push(` --${parameter.replace(/_/g, "-")} '${parameterValue}'`);
+		} else if (typeof parameterValue === "boolean" && parameterValue) {
+			cmd.push(` --${parameter.replace(/_/g, "-")}`);
+		}
+	}
+
+	return cmd;
+}
+
+export async function startExegolContainer(
+	terminal: String,
+	name: String,
+	profileNewContainer: String,
+	jsonProfileData: any,
+) {
+	if (profileNewContainer && jsonProfileData) {
+		var cmdFromProfile = constructCommandFromExegolProfiles(
+			jsonProfileData,
+			profileNewContainer,
+		);
+		var startCmd = `exegol start ${cmdFromProfile.join("")} ${name} ${jsonProfileData[profileNewContainer]["image"]}`;
```

Comment:
you should check if `exegol` is installed and if not, show an helpful error message to the user upon starting the extension to let them know how they can install it

Reply:
<img width="781" height="487" alt="image" src="https://github.com/user-attachments/assets/820221b7-688d-4260-9078-41eba31c8c34" />